libraries/esp8266/examples/NTP-TZ-DST/NTP-TZ-DST.ino

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
Need a header here explaining what the example does, and the if/endif choices below. Also should include a public domain or cc0 license (like the original Arduino examples).

Reply:
The file also needs to be formatted same way as other sketches. Simple way to do this is to load it into arduino, then use Tools > Format sketch.